File: C Code/Basics/Others/Check Prime.c
#include<stdio.h>
#include<math.h>
#include<stdbool.h>
int main(){
  int n;
  printf("Enter Number : ");
  scanf("%d",&n);
  bool isPrime=true;
  for(int i=2;i<=sqrt(n);i++){
    if(n%i==0)
      isPrime=false;
  }
  if(isPrime)
    printf("%d is a Prime Number.\n",n);
  else
    printf("%d is a not Prime Number.\n",n);
  return 0;
}

File: Java Code/Basics/Recursion/CheckPrime.java
import java.util.Scanner;

public class CheckPrime{
  public static boolean isPrime(int n, int i){
    if(i==n)
      return true;
    if(n%i==0)
      return false;
    return isPrime(n,i+1);
  }
  public static void main(String[] args){
    Scanner sc=new Scanner(System.in);
    System.out.print("Enter N : ");
    int n=sc.nextInt();
    if(isPrime(n,2))
      System.out.println(n+" is a prime number.");
    else
      System.out.println(n+" is not a prime number.");
  }
}